app/lib/feed_manager.rb

```diff
@@ -145,19 +145,20 @@ def push_update_required?(timeline_id)
     redis.exists("subscribed:#{timeline_id}")
   end
 
+  def blocks_or_mutes?(receiver_id, account_ids, context)
+    Block.where(account_id: receiver_id, target_account_id: account_ids).any? ||
+      (context == :home ? Mute.where(account_id: receiver_id, target_account_id: account_ids).any? : Mute.where(account_id: receiver_id, target_account_id: account_ids, hide_notifications: true).any?)
```

Comment:
```
Block.where(account_id: receiver_id, target_account_id: account_ids) ||
Mute.where(account_id: receiver_id, target_account_id: account_ids, hide_notifications: context == :notifications)
```

?

Reply:
@nightpool No that will query `WHERE hide_notifications = FALSE` whereas the real query should be `WHERE hide_notifications = ANY` but that's not an indexed column so it's cheaper to omit the clause at all than do it with an `IN` or `OR`